packages/nouns-contracts/contracts/NounsArt.sol

```diff
@@ -327,6 +327,154 @@ contract NounsArt is INounsArt {
         emit GlassesAdded(imageCount);
     }
 
+    /**
+     * @notice Replace current batch of body images with new ones.
```

Comment:
@EricDevito I believe this is inaccurate and worth fixing in all natspecs. In other functions like `addHeads` or `addHeadsFromPointer` we say `Add a batch` in the natspec, because you are indeed adding a new batch into the Trait's dataset, which consists of multiple batches or pages.

In your code you are replacing the entirety of the trait data with this new data. This is an important distinction worth emphasizing.

wdyt?